.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+# Run `pre-commit autoupdate` to update tool versions.
+# Note that pre-commit.ci can update hook versions automatically.
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks.git
+    rev: v4.4.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+      - id: trailing-whitespace
+
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.10.1
+    hooks:
+      - id: pyupgrade
+        name: "Enforce Python 3.6 idioms"
```

Comment:
Wouldn't this be counter to the desire to maintain compatibility with 2.7?

Reply:
Support for Pythons less than 3.6 was dropped ~2 weeks ago.